content/guides/guides/cross-origin-testing.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Cross Origin Testing
+---
+
+<Alert type="info">
+
+<strong class="alert-header"> Note </strong>
+
+As of Cypress [v12.0.0](https://on.cypress.io/changelog#12-0-0), Cypress has the
+capability to visit multiple origins in a single test via the
+[cy.origin()](https://on.cypress.io/origin) command!
+
+</Alert>
+
+Cypress limits each test to visiting domains that share the same superdomain. If
+a navigation occurs that does not meet the same superdomain rule, the
+[`cy.origin`](/api/commands/origin) command must be used to execute assertions
+inside the newly navigated origin.
+
+But what is same superdomain? It is actually very similar to that of same
+origin! Two URLs have the same origin if the protocol, port (if specified), and
+host are the same for both. Cypress automatically handles hosts of the same
```

Comment:
```suggestion
host match. Cypress automatically handles hosts of the same
```

Reply:
> Cypress automatically handles hosts of the same

IMO I think you should open this guide with this statement of how cypress works default then go into what is as uper domain and where origin comes into play

Reply:
I almost feel like you need the opposite in order to understand why cypress works the way it does by default? But I also feel like it takes a minute to get into it 😅

Reply:
got the suggestion applied in a1c9e8b71